### PR TITLE
Validators and Jobs for Lobster

### PR DESCRIPTION
**Pull request by JaGeo**
Hi!

We (@gpetretto +I) are working on integrating Lobster workflows into atomate. Therefore, we would like to include new validators and jobs for Lobster into custodian.

We have also written tests. Please let me know if we should do any changes.

Best, 
JG

### CODE REVIEW

File: custodian/lobster/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+
+"""
+This package implements various VASP Jobs and Error Handlers.
```
Comment (shyuep) [inline custodian/lobster/__init__.py:4]: Change this doc to say lobster?

File: custodian/lobster/handlers.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import os
```
Comment (shyuep) [inline custodian/lobster/handlers.py:1]: There needs to be a module doc.

File: custodian/lobster/jobs.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import logging
```
Comment (shyuep) [inline custodian/lobster/jobs.py:1]: Module doc.

File: test_files/lobster/vasp_lobster_output/FW_submit.script [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (shyuep) [inline test_files/lobster/vasp_lobster_output/FW_submit.script:1]: I don't think this file should be in here.

File: test_files/lobster/vasp_lobster_output/FW.json [hunk 5]
```diff
@@ -0,0 +1,136 @@
+{
```
Comment (shyuep) [inline test_files/lobster/vasp_lobster_output/FW.json:1]: Is this file even necessary. Custodian stands separate from Fireworks.

File: test_files/lobster/lobsterouts/lobsterout.crash [hunk 6]
```diff
@@ -0,0 +1,42 @@
+LOBSTER v3.2.0 (g++ 5.4.0)
```
Comment (shyuep) [inline test_files/lobster/lobsterouts/lobsterout.crash:1]: Not sure what this file is. Or the .keep file.

File: test_files/lobster/vasp_lobster_output/FW_submit.script (hunk 4 above)
Comment (JaGeo) [reply test_files/lobster/vasp_lobster_output/FW_submit.script:1]: will remove this.

File: test_files/lobster/vasp_lobster_output/FW.json (hunk 5 above)
Comment (JaGeo) [reply test_files/lobster/vasp_lobster_output/FW.json:1]: will remove this as well.

File: test_files/lobster/lobsterouts/lobsterout.crash (hunk 6 above)
Comment (JaGeo) [reply test_files/lobster/lobsterouts/lobsterout.crash:1]: will remove the .crash file - the .keep was there to create the folder in the first place. I need the empty folder for the test.

File: custodian/lobster/__init__.py (hunk 1 above)
Comment (JaGeo) [reply custodian/lobster/__init__.py:4]: okay

File: custodian/lobster/handlers.py (hunk 2 above)
Comment (JaGeo) [reply custodian/lobster/handlers.py:1]: okay

File: custodian/lobster/jobs.py (hunk 3 above)
Comment (JaGeo) [reply custodian/lobster/jobs.py:1]: okay

File: test_files/lobster/lobsterouts/lobsterout.crash (hunk 6 above)
Comment (JaGeo) [reply test_files/lobster/lobsterouts/lobsterout.crash]: Do you have a suggestion how I can add an empty folder to git without the .keep file? Thanks!

Comment (shyuep) [reply test_files/lobster/lobsterouts/lobsterout.crash]: This is fine. But why do you need an empty folder for the test?

Comment (JaGeo) [reply test_files/lobster/lobsterouts/lobsterout.crash]: You are correct. I just need a folder without a "lobsterout" file to check that the validator is working. I will use one without lobsterout instead. Thanks.

**Comment by JaGeo** [comment]
Is there anything else I should do? 

Best,
JG

**Comment by shyuep** [comment]
Thanks.

**Comment by JaGeo** [comment]
Thank you very much. I forgot to ask: Would it be possible to release a new custodian version?

**Comment by shyuep** [comment]
Done.

**Pull request merged**